.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+---
+name: "Release"
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  distribute:
+    name: Distribute
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Check out repository'
+        uses: actions/checkout@v2
+      - name: 'Publish the release to GitHub'
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Tsunami ${{ github.ref }}
+          draft: false
+          prerelease: false
+      - name: 'Login to Docker Hub'
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: 'Build and push the docker image'
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: google/tsunami-security-scanner:${{ github.ref }}
```

Comment:
At a quick look, at least [one other google project](https://hub.docker.com/r/google/cadvisor/tags?page=1&ordering=last_updated) uses this format for docker hub tags

Reply:
Thanks for checking!